cea/optimization/master/summarize_network.py

```diff
@@ -11,9 +11,9 @@
 import pandas as pd
 import math
 
-__author__ = "Jimeno A. Fonseca"
-__copyright__ = "Copyright 2015, Architecture and Building Systems - ETH Zurich"
-__credits__ = ["Jimeno A. Fonseca", "Thuy-An Nguyen", "Tim Vollrath", ]
+__author__ = "Sreepathi Bhargava Krishna"
```

Comment:
it seems all you did was change variable names? that doesn't constitute authorship.